docs/source/model_design/vivarium_features/custom_parameterized_draw/index.rst

```diff
@@ -0,0 +1,138 @@
+
+Relative to the parameter's full set of draws or specified uncertainty interval, these requests may take the form of:
+
+- Mean
+- Median
+- Minumum or maximum
+- Some percentile (0.025, 0.1, 0.9, 0.975, etc.)
+- Some specified point value (either within or outside of existing distribution)
+
+Potential use cases
+~~~~~~~~~~~~~~~~~~~
+
+A single draw that uses the average value for all modeled parameters could be used throughout verification and validation, reducing computational resource requirements.
+
+A single draw that uses the average value for all modeled parameters could be used to report an estimate of the average simulation results rather than the average of several randomly sampled draws when computational resources are extremely limited.
+
+Allows for sensitivity analysis around parameter influence (e.g. `tornado diagrams <https://en.wikipedia.org/wiki/Tornado_diagram>`_), which can serve as additional simulation findings and sights of our models. This may allow us to make predictions about how simulation results may differ in similar locations to the modeled location that vary with respect to certain key influential parameters.
+
+Challenges to consider
+~~~~~~~~~~~~~~~~~~~~~~
+
+Some parameter values may not be independent from other parameter values. Examples include:
+
+- Categorical risk exposures must always sum to 1 across categories
+
+- Population attributable fractions are a function of relative risk and risk exposure values
+
+- The all-cause mortality rate is a function of cause-specific mortality rates
+
+- Varying parameter values for younger age groups may have implications for older age groups that could cause lack of validation with respect to these parameters in the older age groups.
+
+- Varying incidence and/or remission rates may make them no longer compatible with prevalence (and other possibilities for similar discontinuities)
+
+There are not guarantees that the simulation results using the average value for all parameters approximates the average simulation result across many randomly sampled draws.
+
+Selecting the "average" value for a parameter may be challenging in the case of skewed parameter distributions such as relative risks where the mean and median values differ.
+
+For sensitivity analysis and tornado diagrams
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Our simulations contain a high number of parameters, so running multiple values for each parameter may be onerous and require significant computational resources.
+
+Identifying reasonable upper and lower bounds for each parameter may be difficult. For instance,
+
+- Selecting upper and lower 95% uncertainty intervals may bias our assessment of influential parameters towards parameters with more parameter uncertainty.
```

Comment:
Whether or not this is a bias depends on what you are using the tornado diagram for, I think. For a standard sensitivity analysis, this seems exactly right and not biased at all. It answers the question: "how does our uncertainty about the inputs contribute to our uncertainty about the output?"

If you want to use the tornado diagram for more of an "effect size" and making preliminary guesses about how things vary by location/sub-population, instead of UIs here you'd want to use some measure of how much you think that parameter varies between locations/sub-populations.

I could imagine a parameter where we're fairly unsure of the true value but for biological reasons we think it is likely to be almost constant across sub-populations; in that case, the two approaches would significantly differ.

Reply:
Good point! How do you feel about the term "will favor" instead of bias? Something like...

`Selecting upper and lower 95% uncertainty intervals will favor parameters with greater parameter uncertainty as more influential parameters. These values may be appropriate if the aim is to assess how input parameter uncertainty influences output uncertainty. However, if the goal is to assess parameter influence across a range of plausible values (across locations or subpopulations, for example), the upper and lower uncertainty bounds may not be the most appropriate parameter values to select.`

Reply:
Sounds good!